app/uk/gov/hmrc/soletraderidentificationfrontend/views/capture_address_page.scala.html

```diff
@@ -130,6 +121,17 @@
             })
         </script>
 
+        @inputText(
+            form = form,
+            id = "postcode",
+            name = "postcode",
+            label = messages("enter-address.postcode"),
+            isPageHeading = false,
+            hasFieldTitle = true,
+            hint = Some(Html(s"""<p class="govuk-hint">${messages("enter-address.hint")}</p>""")),
```

Comment:
This could be simplified to hint = Some(Html(messages("enter-address.hint"))), as the class is automatically added

Reply:
didnt realise class is already added. it's done